django_adobesign/tests/test_adobe_backend.py

```diff
@@ -70,12 +76,44 @@ def test_create_signature(mocker, minimal_signature, adobe_sign_backend):
                         return_value={'transientDocumentId': 'doc_id'})
     mocker.patch.object(AdobeSignClient, 'post_agreement',
                         return_value={'id': 'test_agreement_id'})
+    mocker.patch.object(
+        AdobeSignClient, 'get_members',
+        return_value={
+            'participantSets': [
+                {'memberInfos': [{'email': 'pouet@plop.com'}],
+                 'id': 'fooid',
+                 'status': 'NOT_YET_VISIBLE',
+                 'order': 2},
+                {'memberInfos': [{'email': 'poney@plop.com'}],
+                 'id': 'barid',
+                 'status': 'CANCELLED',
+                 'order': 1}]})
+
+    # Signers
+    signer1 = Signer(full_name='Poney poney', email='poney@plop.com',
+                     signing_order=1)
+    signer2 = Signer(full_name='Pouet pouet', email='pouet@plop.com',
```

Comment:
Why am I thinking of an hilarious version of https://en.wikipedia.org/wiki/Pony_Pony_Run_Run now ?

Reply:
:) we can change pouet pouet with run run ;)